hi-ml-histopathology/src/histopathology/configs/classification/custom_subset/dump_dataset_csv.py

```diff
@@ -0,0 +1,43 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import argparse
```

Comment:
This file and the custom subset files should also move to a different module, maybe in the test folder.